content/en/project/tulpa/index.md

```diff
@@ -0,0 +1,329 @@
+
+Tulpas are not viewed as hallucinations but rather as fully-fledged conscious entities sharing the same neurological space as the host. The creation and interaction with Tulpas require intense concentration and visualization, often leading to the creation of a mentally constructed realm, called a "wonderland," where Tulpamancers and their Tulpas can interact.
+
+Tulpamancy provides a unique framework to explore the capacities of the human mind in creating and perceiving conscious entities. This has substantial implications for our understanding of consciousness, agency, and the malleability of perception. Moreover, it brings forward questions about the neurological underpinnings of these phenomena. How does the brain of a Tulpamancer accommodate multiple conscious entities? Can we observe a distinct neurophysiological pattern associated with the presence of a Tulpa
+
+Through neuroimaging, we aim to investigate these questions, seeking to shed light on the intriguing practice of Tulpamancy and its implications for our understanding of the human mind and consciousness.
+
+To get a better understanding of Tulpamancy you can check the following [video](https://www.youtube.com/watch?v=f3nf6NOPyr8). The video summarises the most important tipps for creating tulpas. These tipps are sourced from the reddit thread [`r/tulpas`](https://www.reddit.com/r/Tulpas/).
+
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=f3nf6NOPyr8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Data
+
+The data used for this project was collected by Dr. Michael Lifshitz (McGill) from 2020 to 2022 at Stanford University, California, USA. The dataset comprises fMRI scans of 22 expert Tulpamancers. The data was aquired during a sentence completion task originally designed by [Walsh et al. (2015)](https://www.sciencedirect.com/science/article/pii/S0010945214003037?casa_token=rzUA8Ep2LcEAAAAA:W-24nUu1MMLfuAMWAIhnP9p_pxmfo1Me0QBdhFrxqeEJ29sWT7pi-7CtrsWeVCBt47AvVjDxowVl). The tasks consists of 10 runs. During each run, participants are given a beginning of a sentnece such as "in summer...". Participants then have 9 seconds to complete this sentence in their mind (preparation) and another 12 seconds to write down the complete sentence on a sheet of paper in front of them (write). These 10 runs are repeated multiple times for different conditions. The conditions comprise self: completing or writing the sentence as yourself; Tulpa: letting the tulpa complete or write the sentence; and a control condition friend: imagine a friend completing or writing the sentence.
+
+<div style="text-align: center; background-color: white; border: 1px solid #000; padding: 20px;">
+   <img src="img/sc_protocol.png" height="250px;" alt=""/>
+   <img src="img/MRI.png" height="250px;" alt=""/>
+   <br /><sub><b>protocol of the sentence completion task</b></sub>
+</div>
```

Comment:
@FrancoisPgm, I moved the picture from the results section here and added a short description to contextualiuze it.

Reply:
@jonasmago regarding (1) it is probably because you didn't installed the theme with `git submodule update --init --recursive --remote` as specified in the readme of the repo.

Reply:
Great, that works. Sorry for missing that.

Reply:
Great. I now also fixed the youtube video!
Only the links to different sections still don't work and I'm not exactly sure why. 
I used the link format that works in .md files, however, they seem not to work in HTML. 

Is there an easy way of doing this?

In HTML it would look something like this:
<a href="#section1">Go to Section 1</a>
<h2 id="section1">Section 1</h2>

But I wonder if there are wasier ways in the webiste format here? 

Otherwise I may also just remove them. 

Jonas

Reply:
Regarding (3), I find it suspicious to have a super high p value for the writing condition (0.32) and a null one for tthe prep condittion, although the accuracy for the writing condition is much higher than for the prep. Also you say "In both conditions, we have an accuracy of ~51%" but for the prep condition you report an average accuracy of 0.37.

Reply:
For the in-page links I would suggest to try some good old fashion HTML : [see here for in-page anchors](https://www.w3docs.com/snippets/html/how-to-create-an-anchor-link-to-jump-to-a-specific-part-of-a-page.html)

Reply:
@FrancoisPgm, you were right with the p-value. 
It's the prep-write condition where p=0.00 (I thought that's the one you linked). 
I just checked the other two and they were mixed up and should be not 0. 
For write (higher accuracy) the p-value is 0.02, for prep (lower accuracy), p=0.32. 

I have now fixed both these values.

Reply:
I've also fixed the links and tested the page locally, I think it runs well. 
Let me know if there ar any other things you spot. Otherwise happy for it to be merged.